peer_mentoring/groups/views.py

```diff
@@ -14,6 +14,13 @@ def group_index(request):
     groups = Group.objects.all()
     form = CreateGroupForm()
     if request.method == "POST":
+        if "group_id" in request.POST:
+            group_id = request.POST.get("group_id")
+            group = get_object_or_404(Group, id=group_id)
+            group.members.add("request.user")
```

Comment:
Quotes, needs to be the object. Already addressed in yesterday's session + pull request.